### DISCUSSION TITLE
Display country specific characters (umlauts)

### DISCUSSION THREAD

**Discussion by RolfWww** [gsantner/markor#2114] [Q&A] [answered]
Normally I create texts with Notepad++ on the PC as plain text UTF, occasionally also as ANSI file. Then I sync them to my Android mobile using MyPhoneExplorer. So far I have not been able to display the umlauts (Ä Ö Ü ß) correctly in Markor. Then I found request #1693 "Supports UTF-8 file with BOM" and set this flag in the new file dialog. Now the umlauts are displayed correctly with both UTF-8 and UTF-8 BOM in all files. However, I still don't really understand how Markor handles code pages. Is there any documentation on this?

Thanks for hints, Rolf

**Comment by gsantner** [comment, marked as answer]
Markor uses generic Java String UTF-8 file reafing/writing. Nothing special.

For BOM, it's mainly about setting/preservimg the info. That's about it.
